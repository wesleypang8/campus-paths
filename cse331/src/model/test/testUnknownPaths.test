#test unknown start/end locations for path

#unknown start
r
BOO
CSE

#unknown end
r
CSE
BOO

#both unknown
r
OOG
BOO

# All of your test scripts should end with q to terminate the program.
q