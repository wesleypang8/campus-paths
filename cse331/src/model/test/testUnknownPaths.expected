Menu:
    r to find a route
    b to see a list of all buildings
    q to quit

Enter an option ('m' to see the menu): #test unknown start/end locations for path

#unknown start
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: BOO

Enter an option ('m' to see the menu): 
#unknown end
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: BOO

Enter an option ('m' to see the menu): 
#both unknown
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: OOG
Unknown building: BOO

Enter an option ('m' to see the menu): 
# All of your test scripts should end with q to terminate the program.
